api/udpa/data/orca/v1/orca_load_report.proto

```diff
@@ -14,22 +14,22 @@ import "validate/validate.proto";
 
 message OrcaLoadReport {
   // CPU utilization expressed as a fraction of available CPU resources. This
-  // should be derived from a sample or measurement taken during the request.
+  // should be derived from the latest sample or measurement.
   double cpu_utilization = 1 [(validate.rules).double.gte = 0, (validate.rules).double.lte = 1];
 
   // Memory utilization expressed as a fraction of available memory
-  // resources. This should be derived from a sample or measurement taken
-  // during the request.
+  // resources. This should be derived from the latest sample or measurement.
   double mem_utilization = 2 [(validate.rules).double.gte = 0, (validate.rules).double.lte = 1];
 
   // Total RPS being served by an endpoint. This should cover all services that an endpoint is
   // responsible for.
   uint64 rps = 3;
 
-  // Application specific requests costs. Each value may be an absolute cost (e.g.
-  // 3487 bytes of storage) or utilization associated with the request,
-  // expressed as a fraction of total resources available. Utilization
-  // metrics should be derived from a sample or measurement taken
-  // during the request.
-  map<string, double> request_cost_or_utilization = 4;
+  // Application specific requests costs. Each value is an absolute cost (e.g. 3487 bytes of
+  // storage) associated with the request.
+  map<string, double> request_cost = 4;
```

Comment:
FWIW, it makes me uncomfortable that we're making a breaking change to what should be a stable API. I think the only saving grace here is that Envoy isn't implementing it yet and neither is the only known consumer, gRPC. @envoyproxy/api-shepherds for visibility, but I think we can allow this one.

Reply:
I thought because this proto was not in any release yet, this wouldn't count as a breaking change.

Reply:
Nup, we're becoming much stricter about this in https://github.com/envoyproxy/envoy/issues/6271. Again, given the fact that in this case we probably have full knowledge, I'm less concerned, but this is an exception to the rule.